src/is.test.ts

```diff
@@ -46,3 +47,97 @@ describe('isArray(value)', () => {
     expect(isArray(1)).to.equal(false);
   });
 });
+
+// isBoolean(value)
+describe('isBoolean(value)', () => {
+  it('Should be a function', () => {
+    expect(isBoolean).to.be.a('function');
+  });
+
+  it('Should return true when "value" is type boolean', () => {
+    expect(isBoolean(true)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type boolean', () => {
+    expect(isBoolean('This is a string')).to.equal(false);
+  });
+});
```

Comment:
Let's get a bit more thorough and use multiple `expect`s to test different values. Something like:

```suggestion
  it('Should return true when "value" is type boolean', () => {
    expect(isBoolean(true)).to.equal(true);
    expect(isBoolean(false)).to.equal(true);
  });

  it('Should return false when "value" is not type boolean', () => {
    expect(isBoolean('This is a string')).to.equal(false);
    expect(isBoolean(100)).to.equal(false);
    expect(isBoolean([])).to.equal(false);
    expect(isBoolean({})).to.equal(false);
  });
});
```

Same with all the others.